src/coreComponents/constitutive/permeability/ParallelPlatesPermeability.cpp

```diff
@@ -30,8 +30,15 @@ namespace constitutive
 
 
 ParallelPlatesPermeability::ParallelPlatesPermeability( string const & name, Group * const parent ):
-  PermeabilityBase( name, parent )
+  PermeabilityBase( name, parent ),
+  m_updateTransversalComponent( true )
 {
+  registerWrapper( viewKeyStruct::transversalPermeabilityString(), &m_transversalPermeability ).
```

Comment:
naming is a bit confusing, when read it first time though it will be the value for transversal permeability
maybe "useTransversalPermeability"?

Reply:
oh no, it's indeed the value, sorry...

Reply:
yeah, it's the value. If not specified it uses the quadrtic law to compute it like for the other components